### PR TITLE
feat: cleaning of the contact module

### PR DESCRIPTION
**Pull request by sebastianthulin**
This is not a refactor. It's simply a cleanup phase, in order to understand the module better in order to refine the modules functionality and purpose later on.

### CODE REVIEW

File: source/php/Module/Contacts/Contacts.php [hunk 1]
```diff
@@ -124,56 +104,21 @@ public function prepareContacts($contacts)
...
-                if (get_field('address', $this->ID) != "1") {
-                    unset($info['address']);
-                }
-
-                if (get_field('phone', $this->ID) != "1") {
-                    unset($info['phone']);
-                }
-            }
+            //Create full title string
+            $titlePropeties = ['administration_unit', 'work_title'];
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/Contacts.php:115]: Misspelled

File: source/php/Module/Contacts/Contacts.php [hunk 2]
```diff
@@ -124,56 +104,21 @@ public function prepareContacts($contacts)
...
+            $fullTitle = array_filter(array_map(function($key) use ($info) {
+                return $info[$key] ?: false;
+            }, $titlePropeties), function($item) {return $item;});
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/Contacts.php:118]: Same as above

File: source/php/Module/Contacts/views/cards.blade.php [hunk 3]
```diff
@@ -9,32 +9,64 @@
     @endtypography
 @endif
 
-<div class="o-grid" aria-labelledby={{'mod-text-' . $ID .'-label'}}>
+<div class="o-grid o-grid--half-gutter" aria-labelledby={{'mod-text-' . $ID .'-label'}}>
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/views/cards.blade.php:12]: Missing quotes around inside aria-labelledby

File: source/php/Module/Contacts/views/cards.blade.php [hunk 4]
```diff
@@ -9,32 +9,64 @@
...
-                @if(!empty($showImages) && !empty($contact['image']['inlineStyle']))
-                <div class="c-card__image">
-                    <div class="c-card__image-background" alt="{{ $contact['full_name'] }}" style="{{ $contact['image']['inlineStyle'] }}"></div>
+                        {{-- Visiting Address --}}
+                        @includeWhen(!empty($contact['visiting_address']), 'components.visiting')
+                        
+                    @endaccordion
                 </div>
+
+                @if(array_filter([!empty($contact['email']), !empty($contact['phone'])]))
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/views/cards.blade.php:51]: Isnt it better to use @if(!empty($contact['email']) || !empty($contact['phone']))?

File: source/php/Module/Contacts/views/cards.blade.php [hunk 5]
```diff
@@ -9,32 +9,64 @@
...
+
+                    {{-- Other content data --}}
+                    @includeWhen(!empty($contact['other']), 'components.other')
+
+                    @accordion([])
+                        {{-- Opening Hours --}}
+                        @includeWhen(!empty($contact['opening_hours']), 'components.openinghours')
+
+                        {{-- Address --}}
+                        @includeWhen(!empty($contact['address']), 'components.adress')
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/views/cards.blade.php:43]: Misspelled "address"

File: source/php/Module/Contacts/views/cards.blade.php [hunk 6]
```diff
@@ -9,32 +9,64 @@
...
+
+                    <div class="u-border__top--1 u-margin__top--auto u-padding__x--2" style="gap: var(--base, 8px); border-top-color: var(--color-border-divider) !important;">
+
+                        {{-- E-mail --}}
+                        @includeWhen(!empty($contact['email']), 'components.email', ['icon' => 'email'])
+
+                        {{-- Phone --}}
+                        @if (!empty($contact['phone']))
+                            @foreach ($contact['phone'] as $phone)
+                                @include('components.phone', ['icon' => 'phone'])
```
Comment (NiclasNorin) [inline source/php/Module/Contacts/views/cards.blade.php:61]: Should $phone not be passed to the component?

**Review by NiclasNorin — CHANGES_REQUESTED** [review]
Small changes

File: source/php/Module/Contacts/Contacts.php (hunk 2 above)
Comment (sebastianthulin) [reply source/php/Module/Contacts/Contacts.php]: Fixed

File: source/php/Module/Contacts/views/cards.blade.php (hunk 5 above)
Comment (sebastianthulin) [reply source/php/Module/Contacts/views/cards.blade.php:43]: Spelled this way in whole codebase. Wont fix. Will be resolved when converting to component.

File: source/php/Module/Contacts/views/cards.blade.php (hunk 6 above)
Comment (sebastianthulin) [reply source/php/Module/Contacts/views/cards.blade.php]: Won't really matter, but fixed.

**Pull request merged**